.appveyor/build_libedit.bat

```diff
@@ -0,0 +1,59 @@
+@echo on
+setlocal
+
+call "%~dp0set_paths.bat"
+
+cd "%BUILDDIR%"
+
+if exist "%BINARIESDIR%\libedit" (
```

Comment:
This is an interesting use case. It seems this change addresses more than just appveyor CI builds.
Are there special instructions on how developers can build PBS on their windows machine?
Can you post test logs of such a compilation?